src/frontend/components/SøknadsSteg/OmDeg/spørsmål.ts

```diff
@@ -1,4 +1,7 @@
 export enum OmDegSpørsmålId {
+    søkerStatsborgerskap = 'søker-statborgerskap',
```

Comment:
Siden dette ikke er spørsmål vi stiller så blir det kanskje litt misvisende at de ligger som en del av denne enumen 🤔  Vi bruker også stort sett spørsmålsId'er til å sette id i typen ISøknadSpørsmål, men siden disse feltene ikke er et ISøknadSpørsmål ville jeg kanskje bare dratt dem ut i en egen Record som heter noe ala personopplysningerSpråkId'er.

Reply:
Ja, det kan jeg 😊

Reply:
Noe slik?

Reply:
Tenkte egentlig slik fordi ser ingen grunn til at vi trenger en egen id utenom språk på en måte 🤔

`export const omDegPersonopplysningerSpråkId: Record<string, string> = {
    søkerStatsborgerskap: 'omdeg.personopplysninger.statsborgerskap',
    sivilstatus: 'omdeg.personopplysninger.sivilstatus',
    søkerAdresse: 'omdeg.personopplysninger.adresse',
};`

Det er sikkert fint å bare ta denne i bruk med en gang og i Om Deg-steget :D